locations/templates/locations/create_inner.html

```diff
@@ -5,9 +5,30 @@
             <form method="POST" enctype="multipart/form-data">
                 {% csrf_token %}
                 {{ form|bulma }}
+                <p><strong>Race Distribution / Occupancy:</strong></p>
+                <div id="form_set">
+                    {{ occupancies.management_form }}
+                    {% for extra_form in occupancies.forms %}
+                        {{ extra_form.non_field_errors }}
+                        {{ extra_form.errors }}
+                        <div class="no_error">
```

Comment:
I can't find this class in our CSS/Sass, please check if there is some commit missing